crates/types/src/logs/mod.rs

```diff
@@ -320,6 +320,8 @@ where
     Serialize,
     Deserialize,
     Hash,
+    BilrostNewType,
```

Comment:
it'd be cool if we can contribute this upstream this but as an attribute on `bilrost::Message`. For instance
```rs
#[derive(bilrost::Message)]
#[bilrost(transparent)]
struct Offset(u32);
```